test/data/valid.tf

```diff
@@ -16,28 +16,19 @@ resource "instaclustr_cluster" "valid" {
         name = "AWS_VPC"
     }
     rack_allocation = {
-        number_of_racks = 3
-        nodes_per_rack = 1
+        number_of_racks = 5
+        nodes_per_rack = 2
     }
 
     bundle {
         bundle = "APACHE_CASSANDRA"
-        version = "3.11.4"
+        version = "apache-cassandra-3.11.8.ic2"
```

Comment:
[Question, Desired] Not sure if we should be using internally versioned C* versions as part of the test TF files

Reply:
I think we have to, there's already precedent in the provisioning API.